pkgs/applications/graphics/wings/default.nix

```diff
@@ -25,14 +28,35 @@ stdenv.mkDerivation rec {
     find . -type f -name "*.[eh]rl" -exec sed -i 's,wings/src/,../src/,' {} \;
     find . -type f -name "*.[eh]rl" -exec sed -i 's,wings/e3d/,../e3d/,' {} \;
     find . -type f -name "*.[eh]rl" -exec sed -i 's,wings/intl_tools/,../intl_tools/,' {} \;
+    # Remove external dependency rules (such as libigl, eigen, cl)
+    find . -type f -name "Makefile" -exec sed -i 's,libigl: _deps/libigl,,g' {} \;
+    find . -type f -name "Makefile" -exec sed -i 's,eigen: _deps/eigen,,g' {} \;
+    find . -type f -name "Makefile" -exec sed -i 's,cl: _deps/cl,,g' {} \;
+    # The following lines were added during an update to 2.3 due to changes to the Makefile. Chances are
+    # there are better ways to do this, but I'm not aware if there are. PRs welcome.
+    # Remove any targets that depend on these dependencies (e.g., c_src)
+    find . -type f -name "Makefile" -exec sed -i 's,c_src: $(DEPS),c_src:,g' {} \;
```

Comment:
Maybe you can just set `DEPS=` in makeFlags for this one?

Reply:
I completely forgot about this, good catch. I'll see what I can do.